scenarios/triso/README.md

```diff
@@ -0,0 +1,75 @@
+# TRi-structural ISOtropic Fuel
+
+This research folder contains information and analysis regarding the questions
+of:
+1. What are the SWU requirements for TRISO fueled reactor transitions that
+   incorporate the proposed LEU+ to HALEU scheme?
+2. What are the impacts of deployment schemes in NFC scenarios, and what parts
+   are realistic or unrealistic in each?
+
+## Scenarios
+We have investigated several growth scenarios at a low, medium, and high
+scale based on different projections in the literature. The calculations for
+these scenarios can be found in `deployment_calcs`, where there is a notebook
+for each level of growth.
+
+### No Growth
+A 0% demand growth deployment scenario where LWRs retire at end of license or
+at 80 years of operation, whatever is longer. If there are Advanced reactors in
+the scenario, they will replace the decommissioned capacity and meet the demand
+growth starting in 2030.
+
+| Scenario | Demand Growth | Reactors          | Notes |
+|----------|---------------|-------------------|-------|
+| DAN1     | 0%            | LWRs              |       |
+| DAN2     | 0%            | LWRs, Xe-100      |       |
+| DAN3     | 0%            | LWRs, MMR         |       |
+| DAN4     | 0%            | LWRs, MMR, Xe-100 |       |
+
+### Low Growth
+A 1% demand growth deployment scenario where LWRs retire at end of license or
+at 80 years of operation, whatever is longer. If there are Advanced reactors in
+the scenario, they will replace the decommissioned capacity and meet the demand
+growth starting in 2030.
+
+| Scenario | Demand Growth | Reactors          | Notes |
+|----------|---------------|-------------------|-------|
+| DAL1     | 1%            | LWRs, Xe-100      |       |
+| DAL2     | 1%            | LWRs, MMR         |       |
+| DAL3     | 1%            | LWRs, MMR, Xe-100 |       |
+
+### Medium Growth
```

Comment:
That is updated on the branches for those scenarios! Good question though, It's like this because I copy and pasted the table and updated the numbers without changing the scenarios :/